_projects/nested-languages.md

```diff
@@ -24,16 +24,29 @@ tags:
   - Core
   - Bears
 ---
-Multiple programming languages can coexist in a single source file. coala would
+Generally, a single source file contains code for exactly one programming language.
+But multiple programming languages can also coexist in a single source file. coala would
 benefit from the ability to determine the programming language of each part of
-a file. If we have e.g. a PHP file, it is likely that we'd have chunks of html
+a file. If we have example a PHP file, it is likely that we'd have chunks of html
 inside a few of them. We would like coala to be able to still run meaningful
 analysis on those files.
 Other examples include:
 
-* awk inside bash
-* JSON inside Javascript
-* HTML, LaTeX or others inside Jinja2
+* JSON inside Javascript is valid in some cases, for example JSONFormatBear for coala.
+* Web templating - languages such as PHP or JSP files mix code into HTML.
+* Cython, it is a superset of the Python language that additionally supports
+  calling C functions and declaring C types on variables and class attributes.
+* We can call Python code from Java using Jython, which is a Python implementation in Java.
+* reStructuredText and Markdown language specify different language for codeblock.
+
+  For example for documents that have to show snippets in different languages,
+  there’s also a code-block directive that is given the highlighting language directly:
+
+```
```

Comment:
You use triple backslash here.  This is embedding a code block into markdown.  When you create a code block, you can also say which language the code is in.

It is deeply ironic that you havent done that ;-)

Please define the language of this code block, and then add a paragraph after this code block which explains that this file has python embedded in rst embedded in markdown.